packages/sdk/src/types/Products.ts

```diff
@@ -0,0 +1,120 @@
+export interface ProductsCreate {
+	name?: string
+	source_id?: string
+	price?: number
+	attributes?: string[]
+	metadata?: Record<string, any>
+	image_url?: string
+}
+
+export interface ProductsGetResponseSkus {
+	skus?: {
+		object: 'list'
+		total: number
+		data?: {
+			id?: string
+			source_id?: string
+			sku?: string
+			currency?: string
+			price?: number
+			attributes: Record<string, string>
+			created_at?: string
+			object?: string
+		}[]
+	}
+}
+
+export interface ProductsCreateResponse {
+	id: string
+	source_id?: string
+	object: 'product'
+	name?: string
+	price?: number
+	attributes?: string[]
+	created_at: string
+}
+
+export type ProductsGetResponse = ProductsCreateResponse & ProductsGetResponseSkus
+
+export type ProductsUpdate = {
+	name?: string
+	id?: string
+	source_id?: string
+	attributes?: string[]
+	price?: number
+	image_url?: string
+	metadata?: Record<string, any>
+}
+
+export type ProductsUpdateResponse = ProductsCreateResponse & ProductsGetResponseSkus
```

Comment:
Unfortunately update does not return "skus" so please drop the `& ProductsGetResponseSkus`